source/plugins/wakatime/index.mjs

```diff
@@ -20,14 +41,18 @@ export default async function({login, q, imports, data, account}, {enabled = fal
     //Querying api and format result (https://wakatime.com/developers#stats)
     console.debug(`metrics/compute/${login}/plugins > wakatime > querying api`)
     const {data: {data: stats}} = await imports.axios.get(`${url}/api/v1/users/${user}/stats/${range}?api_key=${token}`)
+
+    const projectStats = stats.projects?.map(({name, percent, total_seconds:total}) => ({name, percent:percent / 100, total})).sort((a, b) => b.percent - a.percent)
+    const projects = showOnlyGithubPublicRepos ? await pickOnlyGithubPublicRepos({limit, login, axios:imports.axios, projects:projectStats}) : projectStats.slice(0, limit)
```

Comment:
`projectStats` could be undefined so you probably need to handle it

Since other stats (os/languages/editors) can returns undefined too, I guess to be consistent with them, we want to keep the behaviour of return `undefined` instead of an empty array too